lib/reverse_sentence.rb

```diff
@@ -1,6 +1,40 @@
-# A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n)
```

Comment:
Two questions: : 🙂
- **Time complexity:** what is `n` here?
- **Space complexity:** what is this constant relative to?